ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -239,7 +239,12 @@ if tflite2_support_is_available
   endif
 
   if get_option('tflite2-xnnpack-delegate-support')
-    tflite2_compile_args += '-DTFLITE_XNNPACK_DELEGATE_SUPPORTED'
+    # Check header for xnnpack delegate
+    if cc.has_header('tensorflow2/lite/delegates/xnnpack/xnnpack_delegate.h') or cc.has_header('tensorflow/lite/delegates/xnnpack/xnnpack_delegate.h')
+      tflite2_compile_args += '-DTFLITE_XNNPACK_DELEGATE_SUPPORTED'
+    else
+      warning('There is no header file for xnnpack delegate. Disable it')
```

Comment:
If ```tflite2-xnnpack-delegate-support``` is true, this should be error, not warning.
That's why I'm telling you to use type:'feature' instead of type:'boolean' for your needs.

Anyway, 
```
option('tflite2-xnnpack-delegate-support', type: 'boolean', value: 'true')
```
is strange too. If what you want to is "auto", use 'feature".